### PR TITLE
AvalancheGo@v1.7.18

### PR DESCRIPTION
**Pull request by patrick-ogrady**
This PR updates the avalanchego dependency to v1.7.18 and updates the avalanche-network-runner dependency to this commit: https://github.com/ava-labs/avalanche-network-runner/commit/ee59fae043fce29093e29a7967ca97d7283db9af (used by e2e tests and the update is needed for compatibility with AvalancheGo v1.7.18)

### CODE REVIEW

**Review by darioush — APPROVED** [review]

File: scripts/versions.sh [hunk 1]
```diff
@@ -3,5 +3,5 @@
 # Set up the versions to be used
 subnet_evm_version=${SUBNET_EVM_VERSION:-'v0.2.9'}
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.7.16'}
+avalanche_version=${AVALANCHE_VERSION:-'v1.7.18'}
 network_runner_version=${NETWORK_RUNNER_VERSION:-'v1.2.0'}
```
Comment (ceyonur) [inline scripts/versions.sh:7]: seems current ANR version is not compatible with 1.7.18

File: go.mod [hunk 2]
```diff
@@ -5,12 +5,12 @@ go 1.18
...
 	github.com/VictoriaMetrics/fastcache v1.10.0
 	github.com/ava-labs/avalanche-network-runner v1.2.0
-	github.com/ava-labs/avalanchego v1.7.16
+	github.com/ava-labs/avalanchego v1.7.18
 	github.com/cespare/cp v0.1.0
 	github.com/creack/pty v1.1.18
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckarep/golang-set v1.8.0
-	github.com/ethereum/go-ethereum v1.10.20
+	github.com/ethereum/go-ethereum v1.10.21
```
Comment (ceyonur) [inline go.mod:13]: I don't think we have merged geth 1.10.21 or any recent coreth versions yet. Seems that we're still on 1.10.18:
https://github.com/ava-labs/subnet-evm/blob/master/params/version.go#L36

Comment (patrick-ogrady) [reply go.mod:13]: They are merged. That just wasn't updated.

File: scripts/versions.sh (hunk 1 above)
Comment (felipemadero) [reply scripts/versions.sh:7]: logging lib updates for avago 1.7.17 conflicts with ANR 1.2.0 at E2E compile time (or lint). it is solved in darioush PR by using the latest ANR

**Review by aaronbuchwald — APPROVED** [review]

**Pull request merged**